tonilopezmr/day 3/SpiralNeighbors.kt

```diff
@@ -0,0 +1,105 @@
+package day3
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class SpiralNeighbors {
+
+  private fun sumNeighbors(spiral: Map<Dimens, Int>, dimen: Dimens): Int =
+      (spiral[Pair(dimen.first + 1, dimen.second)] ?: 0) +
+          (spiral[Pair(dimen.first + 1, dimen.second + 1)] ?: 0) +
+          (spiral[Pair(dimen.first, dimen.second + 1)] ?: 0) +
+          (spiral[Pair(dimen.first - 1, dimen.second + 1)] ?: 0) +
+          (spiral[Pair(dimen.first - 1, dimen.second)] ?: 0) +
+          (spiral[Pair(dimen.first - 1, dimen.second - 1)] ?: 0) +
+          (spiral[Pair(dimen.first, dimen.second - 1)] ?: 0) +
+          (spiral[Pair(dimen.first + 1, dimen.second - 1)] ?: 0) +
+          (spiral[Pair(dimen.first, dimen.second)] ?: 0)
```

Comment:
![](https://media.giphy.com/media/ogww893cZgaME/giphy.gif)

Reply:
Hahaha this is for not do a for xD I can't waste my time